src/content/docs/en/recipes/build-forms.mdx

```diff
@@ -146,8 +148,10 @@ Astro pages that are rendered on demand can both display and handle forms. In th
 
     Error messages can be sent back to the client by storing them in an `errors` object and accessing it in the template. 
 
-    ```astro title="src/pages/register.astro" ins={5, 12-22, 41, 46, 51}
+    ```astro title="src/pages/register.astro" ins={2, 7, 14-24, 43, 48, 53}
```

Comment:
``````suggestion
    ```astro title="src/pages/register.astro" ins={7, 14-24, 43, 48, 53}
``````
These look good! But we don't need to mark the prerendering line as "new" because it was already added in the previous step!